test/Todo.md

```diff
@@ -4,8 +4,6 @@ have a link to an open issue/PR, or be obvious. Comments/corrections/additions
 welcome.
 
 Linear memory semantics:
- - test that newly allocated memory (program start and `memory.grow`) is zeroed
- - test that `memory.grow` does a full 32-bit unsigned check for page-size divisibility
```

Comment:
This item doesn't seem to be tested here.

Reply:
This item is outdated. See @rossberg's comment at [here](https://github.com/WebAssembly/spec/pull/784#issuecomment-384210795).